Revert "Timescaledb store plugin"

tom95858 opened the pull request:
Reverts ovis-hpc/ovis#562

wooloo1121 commented:
> Reverts #562

Hi @tom95858, thank you for merge my pull request. But is there any problem that caused this?

morrone commented:
I believe that something got a bit confused in Tom's git processes. This reverts things on a branch that is not consistent with the current state of the OVIS-4 branch, so I think it is safe for me to close this. If there was some reason to revert commit 7c985ad7, titled "Add timescaledb store plugin", we would do that with a single revert commit, not like this.

I am going to close this PR on the assumption that this was part of an earlier attempt to cleanup a mistaken push.

@tom95858, if the state on OVIS-4 is not what you intended, and you are having trouble correcting it, just let me know. I can help with that.

tom95858 commented:
Hi Karin,

No you absolutely did everything that was asked. However, when I initially
merged your request, I forgot to say "squash", so there were 60+ commits on
the top of the tree. So thinking I had time since it was Christmas, I
rebased and squashed your commits and then force-pushed them.
Unfortunately, I had one extra commit in my tree that I had forgotten
about. This is entirely on me.

Thanks for your code and your patience.

@Christopher J. Morrone <morrone2@llnl.gov>  we can either revert that
commit (basically a patch that reverts the patch), or we can simply "fix"
it. I have code that makes it better, but does not fix it. We can discuss
on another thread if you like, but basically it has to do with nnodes,
ncpus, gid, etc... that have a zero value at the time job_prolog event is
delivered... and I'm working through characterizing all of this.


On Mon, Dec 28, 2020 at 6:10 PM Christopher J. Morrone <
notifications@github.com> wrote:

> I believe that something got a bit confused in Tom's git processes. This
> reverts things on a branch that is not consistent with the current state of
> the OVIS-4 branch, so I think it is safe for me to close this. If there was
> some reason to revert commit 7c985ad
> <https://github.com/ovis-hpc/ovis/commit/7c985ad7af7a093f528a729b1cecf298c924a8d4>,
> titled "Add timescaledb store plugin", we would do that with a single
> revert commit, not like this.
>
> I am going to close this PR on the assumption that this was part of an
> earlier attempt to cleanup a mistaken push.
>
> @tom95858 <https://github.com/tom95858>, if the state on OVIS-4 is not
> what you intended, and you are having trouble correcting it, just let me
> know. I can help with that.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ovis-hpc/ovis/pull/567#issuecomment-751912012>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABVTPXBMDAT3EFUFOGA2BTDSXEUCHANCNFSM4VLD755A>
> .
>


-- 
Thomas Tucker, President, Open Grid Computing, Inc.
